<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserSetDlg</class>
 <widget class="QDialog" name="UserSetDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>用户管理</string>
  </property>
  <widget class="QLabel" name="lb_User_UserNamel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_User_Password">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_User_Confirm">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>确认密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_User_UserName">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_User_Password">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_User_pwConfirm">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>220</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_User_OK">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>310</y>
     <width>89</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_User_UserNote">
   <property name="geometry">
    <rect>
     <x>101</x>
     <y>270</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
